src/library/dlgtrackinfomulti.ui

```diff
@@ -437,7 +429,7 @@
        </widget>
       </item>
 
-      <item row="0" column="0">
+      <item row="10" column="0">
```

Comment:
do you know why this was `row="0"` previously?

Reply:
🤷‍♂️ 
items with the same position are stacked, that's why we didn't notice.

Reply:
so the spacer was overlayed on another object somewhere near the top of the table, effectively not existing?

Reply:
yup. that's what I noticed with QLabels when I messed up column/row values.

Reply:
Great. thanks for the clarification then.